### PR TITLE
HLA-1190: New installation instructions and small text changes

### PR DESCRIPTION
**Pull request by s-goldman**
Resolves [HLA-1190](https://jira.stsci.edu/browse/HLA-1190)

This PR updates the installation instructions and makes several changes related to the recent readthedocs restructuring. 

**Checklist for maintainers**
- [x] added entry in `CHANGELOG.rst` within the relevant release section
- [x] updated or added relevant tests
- [x] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [x] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)

[Jenkins test](https://plwishmaster.stsci.edu:8081/blue/organizations/jenkins/RT%2FDrizzlepac-Developers-Pull-Requests/detail/Drizzlepac-Developers-Pull-Requests/46/pipeline/329/)

### CODE REVIEW

File: doc/source/drizzlepac_api.rst [hunk 1]
```diff
@@ -2,6 +2,10 @@ DrizzlePac Interface
 ====================
 .. _drizzlepac_api:
 
+DrizzlePac is written in a hierarchical manner with top-level code in python, and some lower-level code in C. The primary user interface is the astrodrizzle.py ``Astrodrizzle()`` function, which accepts input parameters as variables or as a user-specified configuration (.cfg) file. Single- and Multi-Visit Mosaic creation is handled by hapsequencer.py ``run_hap_processing()`` and hapmultisequencer.py ``run_mvm_processing()``, respectively, which perform additional setup before making calls to ``Astrodrizzle()``. Examples of how to use these functions can be found in the DrizzlePac `Notebook Tutorials <https://github.com/spacetelescope/notebooks/tree/master/notebooks/DrizzlePac>`_.
```
Comment (mdlpstsci) [inline doc/source/drizzlepac_api.rst:5]: Give me a chance to write up a few sentences here that you can rework as appropriate which will give a more up-to-date view of drizzlepac.

**Comment by mdlpstsci** [comment]
@s-goldman 
You should feel VERY free to rework what I have said here.  I just wanted to clarify that Drizzlepac produces three types of products, and they are all now generated during pipeline processing.  This comment is for the drizzlepac_api discussion.

Drizzlepac has evolved over time from not only being 
responsible for creating the calibration pipeline 
drizzled products (i.e., <ipppssoot>_dr[c|z].fits), 
but also to being responsible for the generation of 
the two types of Hubble Advanced Products (HAP).  The 
HAP products are Single Visit Mosaics (SVMs) and 
Multiple Visit Mosaics (MVMs) and the creation of 
these products has been incorporated into the 
calibration pipeline processing.  SVM and MVM 
drizzled products are distingushed from pipeline 
drizzled counterparts and each other through their
filenames, as well as internal contents.

```
SVM: hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppssoo>_dr[c|z].fits)
MVM: hst_skycell_p<PPPP>x<XX>y<YY>_<instr>_<detector>_<filter>_<label>_dr[c|z].fits)
```

where PPPP = 4-digit projection cell number
      XX, YY = 2-digit sky cell numbers


runastrodriz.py is the module to control operation of astrodrizzle to
remove distortion and combine HST images.

runsinglehap.py is the module which controls the single visit mosaic
processing.

runmultihap.py is the module which controls the single visit mosaic
processing.

You can discuss "AstroDrizzle" as being a fundamental module which is the basis for all of the above code.

**Comment by s-goldman** [comment]
> @s-goldman You should feel VERY free to rework what I have said here. I just wanted to clarify that Drizzlepac produces three types of products, and they are all now generated during pipeline processing. This comment is for the drizzlepac_api discussion.
> 
> Drizzlepac has evolved over time from not only being responsible for creating the calibration pipeline drizzled products (i.e., _dr[c|z].fits), but also to being responsible for the generation of the two types of Hubble Advanced Products (HAP). The HAP products are Single Visit Mosaics (SVMs) and Multiple Visit Mosaics (MVMs) and the creation of these products has been incorporated into the calibration pipeline processing. SVM and MVM drizzled products are distingushed from pipeline drizzled counterparts and each other through their filenames, as well as internal contents.
> 
> ```
> SVM: hst_<propid>_<obsetid>_<instr>_<detector>_<filter>_<ipppssoo>_dr[c|z].fits)
> MVM: hst_skycell_p<PPPP>x<XX>y<YY>_<instr>_<detector>_<filter>_<label>_dr[c|z].fits)
> ```
> 
> where PPPP = 4-digit projection cell number XX, YY = 2-digit sky cell numbers
> 
> runastrodriz.py is the module to control operation of astrodrizzle to remove distortion and combine HST images.
> 
> runsinglehap.py is the module which controls the single visit mosaic processing.
> 
> runmultihap.py is the module which controls the single visit mosaic processing.
> 
> You can discuss "AstroDrizzle" as being a fundamental module which is the basis for all of the above code.

I also just got a reply from Jenn on the driz_updates slack channel that we should remove the stuff about HAP and move it to the HAP API (which also needs an intro) and I think she is right. 

If it sounds good to you, I will use this for the HAP API intro and create another drizzlepac interface intro just for runastrodriz.

**Review by mdlpstsci — APPROVED** [review]
After our discussion I am willing to approve this PR even though I do not think the SVM and MVM discussion should fall under the MAST category.  These are calibration pipeline produced products and deserve the same level of discussion given to the use of "drizzlepac (runastrodriz)".  Having said this, I think is fine to wait until the documentation is further fleshed out and to read all the docs for a high-level view.  In any case, kudos on your patience and thoroughness!

**Pull request merged**